common/config/api.go

```diff
@@ -106,6 +110,8 @@ func APIFlags() []cli.Flag {
 
 type APIConfig interface {
 	Port() int
+	AuthToken() string
```

Comment:
[nit] doesn't seem necessary to expose `AuthToken` via the interface as it isn't being consumed anywhere.